src/spsc/mod.rs

```diff
@@ -702,7 +600,7 @@ mod tests {
 
     #[test]
     fn iter() {
-        let mut rb: Queue<i32, U4> = Queue::new();
+        let mut rb: Queue<i32, u16, 4> = Queue::u16();
```

Comment:
This is wrong. `::new` is an alias for `::usize()`.

Reply:
Every change from `Queue::new();` to `Queue::$uxx();` should be checked, because not only we are changing the code but tests too... Whereas we should not change tests as much as possible.

Reply:
Good catch!